src/test/pybind/test_rbd.py

```diff
@@ -1137,6 +1137,25 @@ def test_config_list(self):
             for option in image.config_list():
                 eq(option['source'], RBD_CONFIG_SOURCE_CONFIG)
 
+    def test_config_image_set_and_get_and_remove(self):
+        with Image(ioctx, image_name) as image:
+            for option in image.config_list():
+                eq(option['source'], RBD_CONFIG_SOURCE_CONFIG)
+
+            default_value = image.config_image_get("rbd_request_timed_out_seconds")
+            eq(default_value, '30')
+
+            image.config_image_set("rbd_request_timed_out_seconds", "100")
+            modify_value = image.config_image_get("rbd_request_timed_out_seconds")
+            eq(modify_value, '100')
+
+            image.config_image_remove("rbd_request_timed_out_seconds")
+            default_value = image.config_image_get("rbd_request_timed_out_seconds")
```

Comment:
Shouldn't this return a `KeyError` exception?

Reply:
you are right, this will raise "KeyError: no config conf_rbd_request_timed_out_seconds",thanks.